dataprofiler/profilers/json_decoder.py

```diff
@@ -29,7 +31,7 @@ def get_column_profiler_class(class_name: str) -> BaseColumnProfiler:
     return profiler
 
 
-def load_column_profile(serialized_json: dict) -> BaseColumnProfiler:
```

Comment:
What is the intent behind altering this to be a str?

Reply:
fixed to be dict